### ISSUE TITLE
Confused by type of operator embedding

### ISSUE THREAD

**Issue by doingkong** [apache/incubator-mxnet#5055]
## Environment info
Operating System: CentOS release 6.6 (Final)

Compiler:gcc-4.8.2

Package used (Python/R/Scala/Julia):python

MXNet version:0.9.4

MXNet commit hash (`git rev-parse HEAD`): 3b9d82dd909f9a0bdae3a6177db515ede5c53796

Python version and distribution:2.7

## Code
```
category = mx.symbol.Variable('category')
category_embed = mx.symbol.Embedding(data = category, input_dim = 200,
                                         output_dim = 100, name = "category_embed")
arg_name = category_embed.list_arguments()
out_name = category_embed.list_outputs()
arg_type,out_type,_ = category_embed.infer_type(category=np.int32)
print dict(zip(arg_name, arg_type))
print dict(zip(out_name, out_type))
```

## Output
```
{'category': <type 'numpy.int32'>, 'category_embed_weight': <type 'numpy.int32'>}
{'category_embed_output': <type 'numpy.int32'>}
```
Why the type  of embed_weight is numpy.int32 not numpy.float32?

**Comment by eric-haibin-lin** [comment]
fixed by #5832?

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
